src/SourceBuild/content/eng/tools/tasks/PackageDiff/PackageDiff.Tasks/PackageDiff.Tasks.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsPackable>true</IsPackable>
```

Comment:
Do we need to create a package for the build task? Is the plan to use it directly here as part of the validation process? If so then we wouldn't need a package.